rfcs/proposed/zip_view/README.md

```diff
@@ -0,0 +1,40 @@
+# zip_view Support for the oneDPL Range APIs with C++20
+
+## Introduction
+`std::ranges::zip_view` is a powerful utility that enables developers to combine two or more ranges into a single view,
+where each element is represented as a tuple containing corresponding elements from each input range.
+
+## Motivations
+`std::ranges::zip_view` is a convenient way to combine multiple ranges into a single view, where each element of
+the resulting range is a tuple containing one element from each of the input ranges. This can be particularly
+useful for iterating over multiple collections in parallel. `std::ranges::zip_view` was introduced in C++23,
+but many developers are still using C++20 standard. So, oneDPL introduces `oneapi::dpl::ranges::zip_view`,
+with the same API and functionality as `std::ranges::zip_view`.
+
+### Key Requirements
+`oneapi::dpl::ranges::zip_view` should be:
+- compilable with C++20 version (minimum)
+- API-compliant with `std::ranges::zip_view`
+- random accessible view; the "underlying" views also should be random accessible
```

Comment:
In my understanding our intention was to provide `zip_view `like adaptor (for C++20), which "works (is pipeable)" with C++20 ranges/views and can be passed into oneDPL algorithms, because existing `experimental::zip_view`  "doesn't work (is not pipeable)" with C++20 ranges/views.

Reply:
OK, so there is no intent to make it adaptable to different range categories and usable more widely. It needs to be clear in the documentation & specification.

Reply:
And we should also think of implications. If our `zip_view` cannot be used with ranges that are not random access, could it make developers hesitant to use it, possibly impacting oneDPL usage?